tigerpath/majors_and_certificates/majors/NES_2018.json

```
@@ -0,0 +1,213 @@
+{
+  "type": "Major",
+  "name": "Near Eastern Studies",
+  "code": "NES",
+  "degree": "AB",
+  "year": 2018,
+  "explanation": "NES majors must achieve proficiency through at least the 107-level in a Near Eastern language (Arabic, Hebrew, Persian, or Turkish) and must take eight NES departmental courses. Students write one Junior Paper over the course of their junior year and a Senior Thesis their senior year.",
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-near-eastern-studies",
+    "https://nes.princeton.edu/undergraduate/current-majors/concentration-requirements"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Muhammad Q. Zaman",
+      "email": "mzaman@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Jonathan Gribetz",
+      "email": "gribetz@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "A student who has completed at least one course in the department is eligible to concentrate in Near Eastern studies. This course may be a language class or a course or seminar offered in any of the disciplines covered by the department.",
+      "double_counting_allowed": true,
+      "course_list": [
+        "NES *"
+      ]
+    },
+    {
+      "name": "Language",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Students must complete or pass out of four terms (i.e., through the 107 level) of Arabic, Hebrew, Persian, or Turkish. Students are encouraged, albeit not required, to continue language study at the advanced level and to use their chosen Near Eastern language for senior thesis research.",
+      "course_list": [
+        "ARA 107",
+        "HEB 107",
+        "PER 107",
+        "TUR 107"
+      ]

```

It's normally okay to take a higher level language course as well, as that course has the 107 level as a prerequisite, and would imply that they've passed out of 107.
```suggestion
      "course_list": [
        "ARA 107",
        "ARA 3*",
        "ARA 4*",
        "HEB 107",
        "HEB 3*",
        "PER 107",
        "PER 3*",
        "TUR 107",
        "TUR 3*"
      ]
```